Weranika/project-management-app
# Create, delete, update, show column

1. Description:
In this PR was implemented the next features:  **create, delete, update, show columns**. This scope includes appropriate reducers, handling of the requests to the BE, 2 modal popups, UI for columns. Currently the columns are created **only for the specific** board.
2. How to test:
- get new token here https://rss-app-project-manager.onrender.com/api-docs/
- set this token in /reducers/columnsSlice.tsx
- open _Board_ menu item
3. Additions:
BE can be in 'sleep mode'. It means perhaps you should wait some time for the first request.
